src/ngraph/ngraph_utils.cc

```diff
@@ -0,0 +1,66 @@
+/*******************************************************************************
+* Copyright 2018 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "ngraph_utils.h"
+
+#include <algorithm>
+#include <stdexcept>
+
+#include "nnvm/tuple.h"
+
+using namespace std;
+
+namespace ngraph_bridge {
+
+std::ostream& operator<<(std::ostream& os, const ngraph::Shape& s) {
+  return container_to_debug_stream(os, s);
+}
+
+std::ostream& operator<<(std::ostream& os, const nnvm::TShape& s) {
+  container_to_debug_stream(os, s);
+  return os;
```

Comment:
nitpick:
`return container_to_debug_stream(os, s);`?

Reply:
Doesn't match the other implementation.

Reply:
Oh sweet mother of pearl... I'm going to need to restart CI because of that mistake.